### PR TITLE
Latest Payflex Logo uploaded

### PR DESCRIPTION
**Pull request by hemabadgali**
** Checklist **
- [ ] All icons have a corresponding entry in `db/payment_icons.yml`
- [ ] I have followed the icon guidelines detailed in the [CONTRIBUTING.md](https://github.com/activemerchant/payment_icons/blob/master/CONTRIBUTING.md) file
- [ ] I have optimized the icon with [SVGO](https://jakearchibald.github.io/svgomg/)
- [ ] I am confident that all icons are clear and easy to read/understand
- [ ] I have provided a link to the brand icon’s brand guidelines whenever possible.

If this pull request is not adding new icons, you can remove this checklist.

### CODE REVIEW

**Comment by larouxn** [comment]
👋 Hello @hemabadgali, it looks like there are some failing tests. You can see them under the [Ruby 3 CI](https://github.com/activemerchant/payment_icons/pull/452/checks?check_run_id=2680947126).

**Comment by hemabadgali** [comment]
Thanks for the update @LaNeraNura 
I have removed shortcut link

**Comment by larouxn** [comment]
Let's update the icon ([payflex.svg](https://github.com/activemerchant/payment_icons/blob/f4887614b7e7b531dd489171ada24b63cf702ab7/app/assets/images/payment_icons/payflex.svg)) directly instead of adding a new one.

Confirmed this looks to be the current icon via Payflex's site (https://payflex.co.za/).
Screen Shot 2021-05-27 at 18 14 08

**Comment by hemabadgali** [comment]
Yes Confirmed @larouxn its the same

**Comment by larouxn** [comment]
> Yes Confirmed @larouxn its the same

Okay. Can we make these changes on the pre-existing Payflex icon then? We don't need two different Payflex icons.

**Comment by hemabadgali** [comment]
Yes pls , you can replace existing Logo @larouxn

**Comment by hemabadgali** [comment]
@larouxn  any updates ?

**Comment by larouxn** [comment]
> Yes pls , you can replace existing Logo @larouxn 
> we don't need two different

Sorry, would you be able to replace the old logo with the new one inside this PR?

**Comment by hemabadgali** [comment]
@larouxn I have replaced the old one with new

File: app/assets/images/payment_icons/payflex.svg [hunk 1]
```diff
@@ -1 +1 @@
...
... 14.8a4.88 4.88 0 0 1-.25.55 1.47 1.47 0 0 1-.27.36 1.2 1.2 0 0 1-.34.21A1.33 1.33 0 0 1 21 16h-.26a1 1 0 0 1-.25-.06v-.67l.22.08h.21a.62.62 0 0 0 .37-.11.64.64 0 0 0 .23-.3l.3-.75L20 9.78h.83l1.29 3.62 1.28-3.62h.77zM24.34 10.42v-.64h.75v-.66a1 1 0 0 1 .31-.82 1.31 1.31 0 0 1 .91-.3h.24l.23.05v.63a.65.65 0 0 0-.2 0h-.21a.71.71 0 0 0-.41.11c-.1.06-.14.2-.14.39v.62h.85v.64h-.85v3.79h-.73v-3.81zM27.37 8.09h.73v6.12h-.73zM32 11.08a1.11 1.11 0 0 0-.26-.4 1.2 1.2 0 0 0-.4-.26 1.15 1.15 0 0 0-.5-.1 1.19 1.19 0 0 0-.51.1 1.05 1.05 0 0 0-.39.27 1.14 1.14 0 0 0-.25.39 1.6 1.6 0 0 0-.12.49h2.55a1.36 1.36 0 0 0-.12-.49m.18 2.84a2 2 0 0 1-1.25.38 2.32 2.32 0 0 1-.92-.17 2 2 0 0 1-.66-.48 2 2 0 0 1-.4-.74 3.35 3.35 0 0 1-.15-.93 2.52 2.52 0 0 1 .16-.93 2 2 0 0 1 .43-.72 2 2 0 0 1 1.5-.66 1.85 1.85 0 0 1 1 .25 1.93 1.93 0 0 1 .62.62 2.36 2.36 0 0 1 .33.82 3.61 3.61 0 0 1 .07.85h-3.33a2 2 0 0 0 .07.55 1.33 1.33 0 0 0 .25.47 1.32 1.32 0 0 0 .44.32 1.49 1.49 0 0 0 .62.12 1.22 1.22 0 0 0 .76-.22 1 1 0 0 0 .39-.65h.72a1.77 1.77 0 0 1-.64 1.12" fill="#002751"/><path fill="#002751" d="M32.96 9.78h.93l1.07 1.55 1.1-1.55h.88l-1.53 2.04 1.71 2.39h-.93l-1.23-1.84-1.24 1.84h-.88l1.66-2.33-1.54-2.1z"/></svg>
```
Comment (tauthomas01) [inline app/assets/images/payment_icons/payflex.svg:1]: It's missing a few elements:

- `aria-labelledby="pi-payflex"` attribute (like the previous icon)
- `<title id="pi-payflex">Payflex</title>` element (like the previous account)
- `width="38" height="24"` attribute (like the previous account)

**Review by tauthomas01 — CHANGES_REQUESTED** [review]
Hi,

I left some comments.

Please let me know if there are any questions.

**Comment by hemabadgali** [comment]
@larouxn any updates ?

**Comment by hemabadgali** [comment]
@larouxn any updates on logo

**Comment by larouxn** [comment]
👋 @hemabadgali, I'm still seeing CI issues with the new Payflex icon. Can we address the CI failures? Will probably be able merge soon after CI is green.

```
Failure:
PaymentIconTest#test_Every_payment_icon_SVG_has_a_viewBox_attribute [/home/runner/work/payment_icons/payment_icons/test/unit/payment_icon_test.rb:56]:
{:message=>"The 'payflex' SVG file should have a 'viewBox' attribute on the root <svg> tag"}
```

https://github.com/activemerchant/payment_icons/pull/452/checks?check_run_id=2725656187

**Comment by larouxn** [comment]
```
Failure:
PaymentIconTest#test_Every_payment_icon_SVG_is_38x24_pixels [/home/runner/work/payment_icons/payment_icons/test/unit/payment_icon_test.rb:46]:
{:message=>"The 'payflex' SVG file must be 38 pixels wide"}.
Expected: 38
  Actual: 0
```
https://github.com/activemerchant/payment_icons/pull/452/checks?check_run_id=2856272916

Can see here:
Screen Shot 2021-06-18 at 17 26 28

**Comment by TomLister21** [comment]
Hey @larouxn , please could you share what needs to be updated with the logo?

File: app/assets/images/payment_icons/payflex.svg [hunk 2]
```diff
@@ -1 +1 @@
...
...1.1-.22.17-.35.21-.13.04-.27.07-.44.07-.09 0-.18-.01-.27-.02-.09-.01-.17-.03-.26-.06v-.69c.06.03.14.05.23.08.09.02.16.03.22.03.15 0 .28-.04.39-.11.1-.07.18-.18.23-.31l.31-.77-1.81-4.55h.85l1.34 3.74h.02l1.28-3.74h.8l-2 5.17zM24.52 10.42v-.67h.77v-.68c0-.37.11-.65.32-.85.22-.19.53-.29.94-.29.07 0 .15.01.24.02s.17.03.24.05v.65c-.06-.02-.14-.04-.21-.05-.08-.01-.15-.01-.21-.01-.18 0-.32.04-.42.11-.1.07-.15.21-.15.41v.64h.88v.66h-.88v3.91h-.75v-3.91h-.77zM27.64 8.01h.75v6.32h-.75zM32.44 11.1c-.07-.16-.16-.3-.27-.41-.12-.12-.25-.21-.41-.27-.16-.07-.33-.1-.52-.1s-.37.03-.53.1c-.16.07-.29.16-.4.28-.11.12-.2.26-.27.41-.06.16-.1.32-.12.5h2.63c0-.18-.04-.35-.11-.51m.18 2.94c-.34.26-.77.39-1.29.39a2.3 2.3 0 0 1-.95-.18 1.88 1.88 0 0 1-.68-.5c-.18-.21-.32-.47-.41-.76-.09-.3-.14-.62-.16-.96 0-.35.05-.67.16-.96a2.176 2.176 0 0 1 1.13-1.25c.26-.12.55-.18.86-.18.41 0 .74.08 1.01.25.27.17.48.38.65.64.16.26.27.54.34.85.06.31.09.6.08.88h-3.43c-.01.2.02.39.07.57a1.248 1.248 0 0 0 .71.81c.18.08.4.12.65.12.32 0 .58-.07.78-.22.2-.15.34-.37.4-.67h.74c-.09.52-.31.91-.66 1.17"/><path class="st1" d="M33.42 9.75h.97l1.09 1.61 1.14-1.61h.91l-1.58 2.11 1.77 2.47h-.96l-1.28-1.89-1.27 1.89h-.91l1.71-2.41z"/></svg>
```
Comment (tauthomas01) [inline app/assets/images/payment_icons/payflex.svg:1]: - Please remove unused class `st1` and replace by direct attribute `fill`
- `width` and `height` attributes were missing
- Remove unused svg attributes `version="1.1" id="Layer_1"`

```suggestion
<svg xmlns="http://www.w3.org/2000/svg" width="38" height="24" viewBox="0 0 38 24" xml:space="preserve"><g style="opacity:.5"><path fill="#002751" d="M8.04 8.11zM6.88 15.81a.9.9 0 0 0 1.27 0c.03-.03.05-.06.08-.09-.02.03-.05.06-.08.09a.9.9 0 0 1-1.27 0zM.43 8.34zM8.15 9.47a.9.9 0 0 0 0-1.27c-.03-.03-.07-.06-.1-.09.04.03.07.06.1.09a.9.9 0 0 1 0 1.27zM.69 15.93c.25.16.56.18.83.07-.27.12-.58.09-.83-.07zM1.81 8.2a.9.9 0 0 0-1.27 0c-.04.05-.08.09-.11.14.03-.05.07-.09.11-.14a.9.9 0 0 1 1.27 0zM.76 14.33c.39-.34.81-.62 1.26-.84.4-.19.83-.33 1.26-.42.14-.7.14-1.43 0-2.13-.43-.09-.86-.23-1.26-.42-.45-.21-.87-.49-1.26-.84-.07-.07-.15-.14-.22-.21a.7.7 0 0 1-.13-.17c.04.06.08.12.13.17.07.07.13.14.19.21a3.589 3.589 0 0 1-.19 4.86c.07-.07.15-.14.22-.21zM8.22 15.72c.22-.28.24-.67.08-.97.17.31.14.69-.08.97zM.38 14.76a.898.898 0 0 0 .31 1.17.566.566 0 0 1-.15-.12.898.898 0 0 1-.16-1.05z"/><path fill="#002751" d="M6.67 10.52c-.4.19-.83.33-1.26.42-.14.7-.14 1.43 0 2.13.43.09.86.23 1.26.42.45.21.87.49 1.26.84.07.07.15.14.22.21a.8.8 0 0 1 .15.21c-.04-.07-.09-.14-.15-.21-.07-.07-.13-.14-.19-.21a3.589 3.589 0 0 1 .19-4.86c-.07.07-.15.14-.22.21-.39.34-.81.62-1.26.84zM2.86 9.68c.19.4.33.83.42 1.26.7.14 1.43.14 2.13 0 .09-.43.23-.86.42-1.26.21-.45.49-.87.84-1.26.07-.07.14-.14.21-.22-.07.07-.14.13-.21.19a3.589 3.589 0 0 1-4.65 0c-.07-.06-.14-.12-.21-.19.07.07.14.15.21.22.34.39.62.81.84 1.26zM5.83 14.33c-.19-.4-.33-.83-.42-1.26-.7-.14-1.43-.14-2.13 0-.09.43-.23.86-.42 1.26-.21.45-.49.87-.84 1.26-.07.08-.14.15-.21.22.07-.07.14-.13.21-.19a3.589 3.589 0 0 1 4.86.19c-.07-.07-.14-.15-.21-.22-.34-.39-.62-.81-.84-1.26z"/></g><path fill="#002751" d="M7.93 9.68c.07-.07.15-.14.22-.21a.9.9 0 0 0 0-1.27c-.03-.03-.07-.06-.1-.09a.894.894 0 0 0-.99-.04c-.06.03-.13.08-.18.13-.07.08-.14.15-.21.22-.34.39-.62.81-.84 1.26-.19.4-.33.83-.42 1.26.43-.09.86-.23 1.26-.42.45-.22.87-.5 1.26-.84zM2.02 8.42c-.07-.07-.14-.14-.21-.22a.9.9 0 0 0-1.27 0c-.04.05-.08.09-.11.14a.9.9 0 0 0 .11 1.13c.07.07.15.14.22.21.39.34.81.62 1.26.84.4.19.83.33 1.26.42-.09-.43-.23-.86-.42-1.26-.22-.45-.5-.87-.84-1.26zM8.15 14.54c-.07-.07-.15-.14-.22-.21-.39-.34-.81-.62-1.26-.84-.4-.19-.83-.33-1.26-.42.09.43.23.86.42 1.26.21.45.49.87.84 1.26.07.08.14.15.21.22a.9.9 0 0 0 1.27 0c.03-.03.05-.06.08-.09.22-.28.24-.67.08-.97-.05-.07-.1-.14-.16-.21zM.76 14.33c-.08.07-.15.14-.22.21a.898.898 0 0 0 .15 1.39.905.905 0 0 0 1.12-.12c.07-.07.14-.15.21-.22.34-.39.62-.81.84-1.26.19-.4.33-.83.42-1.26-.43.09-.86.23-1.26.42-.45.22-.87.5-1.26.84zM14.39 11.38c-.06-.2-.14-.39-.26-.54a1.24 1.24 0 0 0-.44-.38c-.18-.09-.39-.14-.63-.14-.25 0-.47.05-.65.15-.18.1-.32.23-.43.39-.11.16-.19.35-.24.55-.05.21-.08.42-.08.63 0 .22.03.44.08.65.05.21.14.39.25.55.12.16.26.29.45.39.18.1.4.15.66.15s.48-.05.65-.15c.17-.1.31-.23.42-.4.11-.17.18-.35.23-.57.05-.21.07-.43.07-.65 0-.22-.02-.43-.08-.63m-2.7-1.63v.62h.02c.12-.25.32-.44.58-.55.27-.12.56-.17.88-.17.35 0 .66.06.92.19s.48.31.66.53c.17.22.31.48.39.77.09.29.13.6.13.92 0 .32-.04.63-.13.92-.09.29-.22.54-.39.76-.17.22-.39.38-.65.51a2.187 2.187 0 0 1-1.3.15c-.14-.02-.28-.06-.41-.12a1.448 1.448 0 0 1-.69-.56h-.02v2.35h-.75V9.75h.76zM18.81 11.97c-.09.07-.22.12-.37.15a6 6 0 0 1-.48.08c-.17.02-.34.04-.51.07-.17.03-.32.07-.46.13s-.25.14-.33.25c-.09.11-.13.26-.13.44a.639.639 0 0 0 .27.52c.08.05.17.09.28.11.11.02.22.04.34.04.25 0 .46-.03.64-.1a1.204 1.204 0 0 0 .68-.59c.05-.12.08-.23.08-.34v-.76zm1.22 2.34c-.13.08-.31.11-.54.11-.19 0-.35-.05-.46-.16-.12-.11-.17-.29-.17-.54-.21.25-.45.43-.72.54-.27.11-.57.16-.89.16-.21 0-.4-.02-.59-.07a1.33 1.33 0 0 1-.48-.22c-.14-.1-.24-.23-.32-.39a1.32 1.32 0 0 1-.12-.59c0-.26.04-.47.13-.64.09-.17.2-.3.35-.4.14-.1.31-.18.5-.23.19-.05.38-.1.57-.13.21-.04.4-.07.59-.09.19-.02.35-.05.49-.09.14-.04.25-.09.34-.17.08-.07.12-.18.12-.32 0-.17-.03-.3-.09-.4a.59.59 0 0 0-.24-.23c-.1-.05-.21-.09-.33-.11-.12-.02-.24-.03-.36-.03-.32 0-.58.06-.8.18-.21.12-.33.35-.35.69h-.75c.01-.28.07-.52.18-.72.11-.19.25-.35.42-.47.18-.12.38-.21.61-.26.23-.05.47-.08.73-.08.21 0 .41.01.62.04.2.03.39.09.55.18.17.09.3.22.4.38.1.16.15.38.15.64v2.35c0 .18.01.31.03.39.02.08.09.12.21.12.06 0 .14-.01.23-.04v.6zM22.35 14.94c-.09.22-.18.41-.26.57-.09.15-.18.28-.28.38-.1.1-.22.17-.35.21-.13.04-.27.07-.44.07-.09 0-.18-.01-.27-.02-.09-.01-.17-.03-.26-.06v-.69c.06.03.14.05.23.08.09.02.16.03.22.03.15 0 .28-.04.39-.11.1-.07.18-.18.23-.31l.31-.77-1.81-4.55h.85l1.34 3.74h.02l1.28-3.74h.8l-2 5.17zM24.52 10.42v-.67h.77v-.68c0-.37.11-.65.32-.85.22-.19.53-.29.94-.29.07 0 .15.01.24.02s.17.03.24.05v.65c-.06-.02-.14-.04-.21-.05-.08-.01-.15-.01-.21-.01-.18 0-.32.04-.42.11-.1.07-.15.21-.15.41v.64h.88v.66h-.88v3.91h-.75v-3.91h-.77zM27.64 8.01h.75v6.32h-.75zM32.44 11.1c-.07-.16-.16-.3-.27-.41-.12-.12-.25-.21-.41-.27-.16-.07-.33-.1-.52-.1s-.37.03-.53.1c-.16.07-.29.16-.4.28-.11.12-.2.26-.27.41-.06.16-.1.32-.12.5h2.63c0-.18-.04-.35-.11-.51m.18 2.94c-.34.26-.77.39-1.29.39a2.3 2.3 0 0 1-.95-.18 1.88 1.88 0 0 1-.68-.5c-.18-.21-.32-.47-.41-.76-.09-.3-.14-.62-.16-.96 0-.35.05-.67.16-.96a2.176 2.176 0 0 1 1.13-1.25c.26-.12.55-.18.86-.18.41 0 .74.08 1.01.25.27.17.48.38.65.64.16.26.27.54.34.85.06.31.09.6.08.88h-3.43c-.01.2.02.39.07.57a1.248 1.248 0 0 0 .71.81c.18.08.4.12.65.12.32 0 .58-.07.78-.22.2-.15.34-.37.4-.67h.74c-.09.52-.31.91-.66 1.17"/><path fill="#002751" d="M33.42 9.75h.97l1.09 1.61 1.14-1.61h.91l-1.58 2.11 1.77 2.47h-.96l-1.28-1.89-1.27 1.89h-.91l1.71-2.41z"/></svg>
```

**Review by tauthomas01 — CHANGES_REQUESTED** [review]

**Comment by larouxn** [comment]
Looks like we're missing a `<title>` attribute like in the previous icon.

```
Failure:
PaymentIconTest#test_Every_payment_SVG_meets_accessibility_requirements [/home/runner/work/payment_icons/payment_icons/test/unit/payment_icon_test.rb:68]:
{:message=>"The 'payflex' SVG file should have a single <title> tag"}.
Expected: 1
  Actual: 0
```

https://github.com/activemerchant/payment_icons/pull/452/checks?check_run_id=2856272916

**Comment by TomLister21** [comment]
Hi @larouxn and @tauthomas01 , could you possibly highlight or show exactly where and what we need to change in the SVG? As our designers are sure they have followed the specs correctly.

**Comment by larouxn** [comment]
👋 Hello @TomLister21

Seeing the following CI error.
```
Failure:
PaymentIconTest#test_Every_payment_icon_SVG_is_38x24_pixels [/home/runner/work/payment_icons/payment_icons/test/unit/payment_icon_test.rb:46]:
{:message=>"The 'payflex' SVG file must be 38 pixels wide"}.
Expected: 38
  Actual: 0
```

Additionally, it looks like the new SVG is much larger and no longer in icon form (white background, curved corners, etc.). We'll need to update accordingly so this new icon is a drop in replacement for the older icon.
Screen Shot 2021-08-03 at 16 20 43

**Comment by TomLister21** [comment]
Hey @larouxn and @tauthomas01 , we noticed the logo failed again. Could you share the reasons this time?

**Comment by larouxn** [comment]
@TomLister21, the current issue is the icon is missing a `<title>` element.

```
Failure:
PaymentIconTest#test_Every_payment_SVG_meets_accessibility_requirements [/home/runner/work/payment_icons/payment_icons/test/unit/payment_icon_test.rb:68]:
{:message=>"The 'payflex' SVG file should have a single <title> tag"}.
Expected: 1
  Actual: 0
```

https://github.com/activemerchant/payment_icons/pull/452/checks?check_run_id=3238056302

**Comment by TomLister21** [comment]
Okay thanks @larouxn . Does the title element have to be something specific?

**Comment by larouxn** [comment]
@TomLister21, should contain a title like the previous icon, `<title id="pi-payflex">Payflex</title>`. Might be able to remove the `id="pi-payflex`. Not entirely sure. Will have to see CI.

**Comment by TomLister21** [comment]
Hi @larouxn , I noticed our logo upload failed the checks again. Is it still the title element issue?

**Comment by tauthomas01** [comment]
Here is the error message

"The 'payflex' SVG file should have a 'viewBox' attribute on the root <svg> tag"

**Comment by larouxn** [comment]
Looks like the updated SVG is missing a `<title>`. Let's add it back in.
https://github.com/activemerchant/payment_icons/pull/452/checks?check_run_id=3719562606

```
Failure:
PaymentIconTest#test_Every_payment_SVG_meets_accessibility_requirements [/home/runner/work/payment_icons/payment_icons/test/unit/payment_icon_test.rb:68]:
{:message=>"The 'payflex' SVG file should have a single <title> tag"}.
Expected: 1
  Actual: 0

rails test /home/runner/work/payment_icons/payment_icons/test/unit/payment_icon_test.rb:61
```

Also these tests can be run locally if you have a Ruby setup with `bundle exec rails test`.

**Comment by larouxn** [comment]
I think something like this is in the right direction (includes all missing attributes) but it's still missing the proper shape/borders.

```svg
<svg aria-labelledby="pi-payflex" width="38" height="24" viewBox="0 0 38 24" fill="none" xmlns="http://www.w3.org/2000/svg" version="1.1" role="img"><title id="pi-payflex">Payflex</title><path fill="#fff" d="M0 0h38v24H0z"/><g opacity=".5"><path style="fill:#002751" d="M8 8.11a.88.88 0 0 0-1 0 .88.88 0 0 1 1 0Zm-1.12 7.7a.9.9 0 0 0 1.27 0l.08-.09-.08.09a.9.9 0 0 1-1.27 0ZM.43 8.34a.9.9 0 0 0 0 1 .9.9 0 0 1 0-1Zm7.72 1.13a.9.9 0 0 0 0-1.27l-.1-.09.1.09a.9.9 0 0 1 0 1.27ZM.69 15.93a.91.91 0 0 0 .83.07.88.88 0 0 1-.83-.07ZM1.81 8.2a.9.9 0 0 0-1.27 0 1.34 1.34 0 0 0-.11.14 1.34 1.34 0 0 1 .11-.14.9.9 0 0 1 1.27 0Z"/><path style="fill:#002751" d="M.76 14.33A5.77 5.77 0 0 1 2 13.49a5.65 5.65 0 0 1 1.26-.42 5.45 5.45 0 0 0 0-2.13A5.65 5.65 0 0 1 2 10.52a5.13 5.13 0 0 1-1.24-.84l-.22-.21a.7.7 0 0 1-.13-.17 1.09 1.09 0 0 0 .13.17l.19.21a3.59 3.59 0 0 1 0 4.65l-.19.21a.82.82 0 0 0-.16.22.82.82 0 0 1 .16-.22Zm7.46 1.39a.9.9 0 0 0 .08-1 .88.88 0 0 1-.08 1Z"/><path style="fill:#002751" d="M.38 14.76a.9.9 0 0 0 .16 1.05l.15.12a.57.57 0 0 1-.15-.12.9.9 0 0 1-.16-1.05Zm6.29-4.24a5.65 5.65 0 0 1-1.26.42 5.45 5.45 0 0 0 0 2.13 5.65 5.65 0 0 1 1.26.42 5.13 5.13 0 0 1 1.26.84l.22.21a.8.8 0 0 1 .15.21 1.37 1.37 0 0 0-.15-.21L8 14.33a3.59 3.59 0 0 1 0-4.65l.19-.21-.22.21a5.77 5.77 0 0 1-1.3.84Zm-3.81-.84a5.65 5.65 0 0 1 .42 1.26 5.45 5.45 0 0 0 2.13 0 5.65 5.65 0 0 1 .42-1.26 5.13 5.13 0 0 1 .84-1.26l.21-.22a.75.75 0 0 1 .18-.13.59.59 0 0 0-.18.13l-.21.19a3.59 3.59 0 0 1-4.67 0l-.19-.19.19.22a5.77 5.77 0 0 1 .86 1.26Zm2.97 4.65a5.65 5.65 0 0 1-.42-1.26 5.45 5.45 0 0 0-2.13 0 5.65 5.65 0 0 1-.42 1.26A5.13 5.13 0 0 1 2 15.59l-.21.22a.85.85 0 0 1-.29.19.91.91 0 0 0 .29-.19l.21-.19a3.59 3.59 0 0 1 4.65 0l.21.19-.21-.22a5.77 5.77 0 0 1-.82-1.26Z"/></g><path style="fill:#002751" d="m7.93 9.68.22-.21a.9.9 0 0 0 0-1.27l-.1-.09a.89.89 0 0 0-1 0 .75.75 0 0 0-.18.13l-.21.22a5.77 5.77 0 0 0-.84 1.26 5.65 5.65 0 0 0-.42 1.26 5.65 5.65 0 0 0 1.26-.42 5.77 5.77 0 0 0 1.27-.88ZM2 8.42l-.19-.22a.9.9 0 0 0-1.27 0 1.34 1.34 0 0 0-.11.14.9.9 0 0 0 0 1 1.09 1.09 0 0 0 .13.17l.22.21a5.77 5.77 0 0 0 1.22.8 5.65 5.65 0 0 0 1.26.42 5.65 5.65 0 0 0-.42-1.26A5.77 5.77 0 0 0 2 8.42Zm6.15 6.12-.22-.21a5.77 5.77 0 0 0-1.26-.84 5.65 5.65 0 0 0-1.26-.42 5.65 5.65 0 0 0 .42 1.26 5.13 5.13 0 0 0 .84 1.26l.21.22a.9.9 0 0 0 1.27 0l.08-.09a.9.9 0 0 0 .08-1Zm-7.39-.21-.22.21a.82.82 0 0 0-.16.22.9.9 0 0 0 .16 1.05l.15.12a.91.91 0 0 0 1.12-.12l.19-.22a5.77 5.77 0 0 0 .84-1.26 5.65 5.65 0 0 0 .42-1.26 5.65 5.65 0 0 0-1.26.42 5.77 5.77 0 0 0-1.24.84Zm13.63-2.95a1.59 1.59 0 0 0-.26-.54 1.24 1.24 0 0 0-.44-.38 1.39 1.39 0 0 0-.63-.14 1.33 1.33 0 0 0-.65.15 1.24 1.24 0 0 0-.43.39 1.7 1.7 0 0 0-.24.55 2.72 2.72 0 0 0-.08.63 2.82 2.82 0 0 0 .08.65 1.63 1.63 0 0 0 .25.55 1.31 1.31 0 0 0 .45.39 1.34 1.34 0 0 0 .66.15 1.28 1.28 0 0 0 .65-.15 1.24 1.24 0 0 0 .42-.4 1.71 1.71 0 0 0 .23-.57 2.81 2.81 0 0 0 .07-.65 2.18 2.18 0 0 0-.08-.63m-2.7-1.63v.62a1.11 1.11 0 0 1 .58-.55 2.13 2.13 0 0 1 .88-.17 2 2 0 0 1 .92.19 2 2 0 0 1 .66.53 2.24 2.24 0 0 1 .39.77 3.32 3.32 0 0 1 0 1.84 2.38 2.38 0 0 1-.39.76 1.82 1.82 0 0 1-.65.51 2.19 2.19 0 0 1-.92.19 1.82 1.82 0 0 1-.38 0 1.48 1.48 0 0 1-.41-.12 1.47 1.47 0 0 1-.69-.56v2.35h-.75V9.75ZM18.81 12a.9.9 0 0 1-.37.15l-.44.05-.51.07a2.06 2.06 0 0 0-.46.13.76.76 0 0 0-.33.25.68.68 0 0 0-.13.44.59.59 0 0 0 .08.31.64.64 0 0 0 .19.21.78.78 0 0 0 .28.11 1.86 1.86 0 0 0 .34 0 1.76 1.76 0 0 0 .64-.1 1.2 1.2 0 0 0 .43-.26 1.11 1.11 0 0 0 .25-.33.88.88 0 0 0 .08-.34V12ZM20 14.31a1 1 0 0 1-.54.11.63.63 0 0 1-.46-.16.7.7 0 0 1-.17-.54 1.85 1.85 0 0 1-.72.54 2.35 2.35 0 0 1-.89.16 2.27 2.27 0 0 1-.59-.07 1.33 1.33 0 0 1-.48-.22 1 1 0 0 1-.32-.39 1.32 1.32 0 0 1-.12-.59 1.35 1.35 0 0 1 .13-.64 1 1 0 0 1 .35-.4 1.53 1.53 0 0 1 .5-.23c.19-.05.38-.1.57-.13l.59-.09a2.87 2.87 0 0 0 .49-.09.84.84 0 0 0 .34-.17.41.41 0 0 0 .12-.32.76.76 0 0 0-.09-.4.59.59 0 0 0-.24-.23 1.17 1.17 0 0 0-.33-.11h-.36a1.63 1.63 0 0 0-.8.18.78.78 0 0 0-.35.69h-.75a1.59 1.59 0 0 1 .18-.72 1.47 1.47 0 0 1 .42-.47 1.76 1.76 0 0 1 .61-.26 3.4 3.4 0 0 1 .73-.08 4.3 4.3 0 0 1 .62 0 1.62 1.62 0 0 1 .55.18 1 1 0 0 1 .4.38 1.21 1.21 0 0 1 .15.64v2.35a1.79 1.79 0 0 0 0 .39q0 .12.21.12a.74.74 0 0 0 .23 0v.6Zm2.35.63c-.09.22-.18.41-.26.57a2 2 0 0 1-.28.38.84.84 0 0 1-.35.21 1.45 1.45 0 0 1-.44.07h-.27a1.22 1.22 0 0 1-.26-.06v-.71l.23.08h.22a.71.71 0 0 0 .39-.11.67.67 0 0 0 .23-.31l.31-.77-1.81-4.52h.85l1.34 3.74 1.28-3.74h.8Zm2.17-4.52v-.67h.77v-.68a1.12 1.12 0 0 1 .32-.85 1.39 1.39 0 0 1 .94-.29h.24L27 8v.65l-.21-.05h-.21a.72.72 0 0 0-.42.11.48.48 0 0 0-.15.41v.64h.88v.66H26v3.91h-.75v-3.92h-.77Zm3.12-2.41h.75v6.32h-.75zm4.8 3.09a1.33 1.33 0 0 0-.27-.41 1.13 1.13 0 0 0-.41-.27 1.27 1.27 0 0 0-.52-.1 1.32 1.32 0 0 0-.53.1 1.15 1.15 0 0 0-.4.28 1.62 1.62 0 0 0-.27.41 2 2 0 0 0-.12.5h2.63a1.26 1.26 0 0 0-.11-.51m.18 2.9a2.06 2.06 0 0 1-1.29.39 2.3 2.3 0 0 1-.95-.18 1.88 1.88 0 0 1-.68-.5 2.14 2.14 0 0 1-.41-.76 4.19 4.19 0 0 1-.16-1 2.69 2.69 0 0 1 .16-1 2.18 2.18 0 0 1 .45-.75 2.1 2.1 0 0 1 .68-.5 2 2 0 0 1 .86-.18 1.85 1.85 0 0 1 1 .25 2.09 2.09 0 0 1 .65.64 2.75 2.75 0 0 1 .34.85 3.89 3.89 0 0 1 .08.88h-3.42a1.78 1.78 0 0 0 .07.57 1.26 1.26 0 0 0 .71.81 1.6 1.6 0 0 0 .65.12 1.26 1.26 0 0 0 .78-.22 1.06 1.06 0 0 0 .4-.67h.74a1.76 1.76 0 0 1-.66 1.25m.8-4.25h.97l1.09 1.61 1.14-1.61h.91l-1.58 2.11 1.77 2.47h-.96l-1.28-1.89-1.27 1.89h-.91l1.71-2.41-1.59-2.17z"/></svg> 
```

**Comment by hemabadgali** [comment]
hi @larouxn its been so long, if you can do the changes and upload that would be great

**Comment by larouxn** [comment]
Pushed a PR to update the Payflex icon over here.
- https://github.com/activemerchant/payment_icons/pull/452

CC @tauthomas01

**Comment by hemabadgali** [comment]
Thanks @larouxn

**Comment by hemabadgali** [comment]
@larouxn when the logo will be available

**Comment by larouxn** [comment]
@hemabadgali, ideally it'll be live this week but it may take until next week. Will let you know when it's live.

Feel free to close this PR when you like. The updated icon has merged in https://github.com/activemerchant/payment_icons/pull/490.

**Comment by larouxn** [comment]
@hemabadgali, ideally it'll be live this week but it may take until next week. Will let you know when it's live.

Closing this PR as it's not longer needed since the updated icon merged in https://github.com/activemerchant/payment_icons/pull/490.

**Pull request closed without merging**